src/map/utils/battleutils.cpp

```diff
@@ -675,7 +675,7 @@ namespace battleutils
 
         ELEMENT spikeElement = (ELEMENT)((uint8)GetSpikesDamageType(Action->spikesEffect) - (uint8)DAMAGE_TYPE::ELEMENTAL);
 
-        int32 damage = 0;
+        int32 damage = Action->spikesParam;
```

Comment:
seems this was zero'd in https://github.com/LandSandBoat/server/pull/1878

unsure if by accident or was meant to have an assignment further down that was lost in a rebase or forgotten

Reply:
unless I hear otherwise: putting param back looks good to me